ui/app/components/clients/running-total.ts

```diff
@@ -4,12 +4,13 @@
  */
 
 import Component from '@glimmer/component';
-import type { ByMonthClients, TotalClients } from 'core/utils/client-count-utils';
+import type { TotalClients } from 'core/utils/client-count-utils';
 import ClientsVersionHistoryModel from 'vault/vault/models/clients/version-history';
+import { ByMonthNewClients } from '../../../lib/core/addon/utils/client-count-utils';
```

Comment:
this type lives in the same directory as TotalClients above, it looks like auto-import just got confused. I also went ahead and added `type` to the import on line 8 🙃 

```suggestion
import type { ByMonthNewClients, TotalClients } from 'core/utils/client-count-utils';
import type ClientsVersionHistoryModel from 'vault/vault/models/clients/version-history';
```